platform.md

```diff
@@ -735,43 +735,50 @@ Usage:
 #### `launcher`
 Usage:
 ```
-/cnb/process/<process-type> [<arg>...]
+/cnb/process/<process-type> [<args>...]
 # OR
-/cnb/lifecycle/launcher [--] [<cmd> <arg>...]
+/cnb/lifecycle/launcher [<cmd> <args>...]
```

Comment:
This is a **very** significant end user change. We should be careful around this since users won't be able to control this. How do you imagine this working in general since processes are global and not buildpack specific? Would processes contributed by the older buildpack apis still allow for the old launcher approach?